### ISSUE TITLE
Graceful FS deprecated warning

### ISSUE THREAD

**Issue by adambuczynski** [cowboy/node-globule#29]
Could you update the glob dependency to the latest, which uses a later version of graceful-fs so to avoid the deprecation warnings?

```
npm WARN deprecated graceful-fs@1.2.3: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
```

```
┬ globule@0.1.0
│       └─┬ glob@3.1.21
│         └── graceful-fs@1.2.3 
```

Thanks!

**Comment by ndaidong** [comment]
+1
Does anyone maintain this repo?

**Comment by jonathanasquier** [comment]
+1

**Comment by mdvorscak** [comment]
```
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
├─┬ gulp@3.9.1 
...
│ └─┬ vinyl-fs@0.3.14 
│   ├─┬ glob-watcher@0.0.6 
│   │ └─┬ gaze@0.5.2 
│   │   └─┬ globule@0.1.0 
│   │     ├─┬ glob@3.1.21 
│   │     │ ├── graceful-fs@1.2.3 
│   │     │ └── inherits@1.0.2 
│   │     ├── lodash@1.0.2 
```
+1

**Comment by mathewmeconry** [comment]
+1

**Comment by shama** [comment]
Fixed on 1c6b40d5b397729be1a79b5737251aa83549e3b0
